docs/managing-templates.md

```diff
@@ -134,3 +134,154 @@ Now let's make use of our new template.
+    ```
+2. Save a copy of the template locally.
+    ```
+    onefuzz job_templates manage get libfuzzer_linux > libfuzzer_linux_ado_areapath.json
+    ```
+3. With your preferred text editor, add the following to the `notifications` list:
+    ```json 
+    {
+      "config": {
+        "base_url": "https://dev.azure.com/org_name",
+        "auth_token": "ADO_AUTH_TOKEN",
+        "type": "Bug",
+        "project": "{{ job.project }}",
+        "ado_fields": {
+          "System.AreaPath": "{{ task.tags['area_path'] }}",
+          "Microsoft.VSTS.Scheduling.StoryPoints": "1",
+          "System.IterationPath": "{{ task.tags['iteration_path'] }}",
+          "System.Title": "{{ report.crash_site }} - {{ report.executable }}",
+          "Microsoft.VSTS.TCM.ReproSteps": "This is my call stack: <ul> {% for item in report.call_stack %} <li> {{ item }} </li> {% endfor %} </ul>"
+        },
+        "comment": "This is my comment. {{ report.input_sha256 }} {{ input_url }} <br> <pre>{{ repro_cmd }}</pre>",
+        "unique_fields": ["System.Title", "System.AreaPath"],
+        "on_duplicate": {
+          "comment": "Another <a href='{{ input_url }}'>POC</a> was found in <a href='{{ target_url }}'>target</a>. <br> <pre>{{ repro_cmd }}</pre>",
+          "set_state": { "Resolved": "Active" },
+          "ado_fields": {
+            "System.IterationPath": "{{ task.tags['iteration_path'] }}"
+          },
+          "increment": ["Microsoft.VSTS.Scheduling.StoryPoints"]
+        }
+      }
+    }
+    ```
+4. With your preferred text editor, add the following to user fields to the end of the `user_fields` list.
+    1. A required field that specifies the tag name `area_path`.
+        ```json
+        {
+            "help": "Area path for reported crashes",
+            "locations": [
+                {
+                    "op": "add",
+                    "path": "/tasks/1/tags/area_path"
+                }
+            ],
+            "name": "area_path",
+            "required": true,
+            "type": "Str"
+        }
+        ```
+    2. An optional field that specifies the `iteration_path`
+        ```json
+        {
+            "default": "Iteration\\Path\\Default\\Here",
+            "help": "Iteration path for reported crashes",
+            "locations": [
+                {
+                    "op": "add",
+                    "path": "/tasks/1/tags/iteration_path"
+                }
+            ],
+            "name": "iteration_path",
+            "required": false,
+            "type": "Str"
+        }
+        ```
+    > NOTE: Each of these fields use the `jsonpatch` path to add a tag to the second task (by array index), which is the `crash_report` task.  Check out [form fields](declarative-templates.md#example-form-fields) for more information.
+5. Upload the template.
+    ```
+    onefuzz job_templates manage upload libfuzzer_linux_ado_areapath @./libfuzzer_linux_ado_areapath.json
+    ```
```

Comment:
Consider reminding the reader of the meaning of `@`. Not strictly necessary, but might be appreciated.